2021-09-09 13:01:15,851 INFO    MainThread:40916 [internal.py:wandb_internal():86] W&B internal server running at pid: 40916, started at: 2021-09-09 13:01:15.841024
2021-09-09 13:01:15,857 INFO    WriterThread:40916 [datastore.py:open_for_write():77] open: C:\Programming\rl_cooperation\Pygame_RL_cooperation\Algorithm\DQN_SpaceCannons_PER\wandb\offline-run-20210909_130114-dhewokp4\run-dhewokp4.wandb
2021-09-09 13:01:15,858 DEBUG   HandlerThread:40916 [handler.py:handle_request():126] handle_request: run_start
2021-09-09 13:01:15,877 DEBUG   HandlerThread:40916 [meta.py:__init__():40] meta init
2021-09-09 13:01:15,878 DEBUG   HandlerThread:40916 [meta.py:__init__():54] meta init done
2021-09-09 13:01:15,878 DEBUG   HandlerThread:40916 [meta.py:probe():211] probe
2021-09-09 13:01:15,965 DEBUG   HandlerThread:40916 [git.py:repo():33] git repository is invalid
2021-09-09 13:01:15,965 DEBUG   HandlerThread:40916 [meta.py:_save_pip():58] save pip
2021-09-09 13:01:15,966 DEBUG   HandlerThread:40916 [meta.py:_save_pip():72] save pip done
2021-09-09 13:01:15,966 DEBUG   HandlerThread:40916 [meta.py:probe():249] probe done
